microprofile/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/resolver/JaegerTracerConfiguration.java

```diff
@@ -102,6 +100,11 @@ public String getName() {
 
     @Override
     public Tracer createTracer(String serviceName) {
+        Configuration.SenderConfiguration senderConfiguration = reporterConfig.getSenderConfiguration();
+        if(outboundSocketBindingSupplier.get() != null) {
+            senderConfiguration.withAgentHost(outboundSocketBindingSupplier.get().getUnresolvedDestinationAddress())
+                    .withAgentPort(outboundSocketBindingSupplier.get().getDestinationPort());
+        }
```

Comment:
I don't really know how this is used, but we might want to do something more like.
```
final OutboundSocketBinding binding = outboundSocketBindingSupplier.get();
if(outboundSocketBindingSupplier.get() != null) {
    senderConfiguration.withAgentHost(binding.getUnresolvedDestinationAddress())
            .withAgentPort(binding.getDestinationPort());
}
```

I don't know what the overhead of `get()` is in MSC, but it seems like a single `get()` would be better.